CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- votes showing wrong amount @faboweb
+- depositing less then 1 atom not possible @faboweb
```

Comment:
uh why not ?

Reply:
form validation prevented that, I fixed it